e2eshark/run.py

```diff
@@ -333,8 +324,12 @@ def runCodeGeneration(
         print(f"Test {testName} failed [{curphase}]")
         return 1
     logfilename = curphase + ".log"
-    commandname = (
-        "/tools/iree-compile --iree-input-demote-i64-to-i32 --iree-hal-target-backends="
+    commandname = ""
+    if SHARED_IREE_BUILD:
+        commandname = SHARED_IREE_BUILD + "/tools/"
```

Comment:
Binary (non-Python) distributions are also useful to support. Those don't include a `tools` folder (they are flatter)

e.g.
```
iree-dist/
  bin/
    iree-compile
    iree-run-module
  include/
    iree/...
      (header files)
  lib/
    libIREECompiler.so
```

The most generic way to support both is to just point at the binary directory, either `bin/` from an install/dist like that or `tools/` from a build directory